HeterogeneousTest/AlpakaDevice/interface/alpaka/DeviceAddition.h

```diff
@@ -0,0 +1,37 @@
+#ifndef HeterogeneousTest_AlpakaDevice_interface_alpaka_DeviceAddition_h
+#define HeterogeneousTest_AlpakaDevice_interface_alpaka_DeviceAddition_h
+
+#include <cstdint>
+
+#include <alpaka/alpaka.hpp>
+
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/workdivision.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE::test {
```

Comment:
Given that the `add_vectors_f` and `add_vectors_d` are templated over `TAcc`, I guess the `ALPAKA_ACCELERATOR_NAMESPACE` is not strictly required here?

Reply:
Good point.

The CUDA/ROCm tests are in the namespace `cms::cudatest`/`cms::rocmtest` to avoid conflicts with other packages.

Shall I use `cms::alpakatest` here ?